src/adr/0025-cassandra-blob-store-cache.md

```diff
@@ -0,0 +1,66 @@
+# 25. Cassandra Blob Store Cache
+
+Date: 2020-04-03
+
+## Status
+
+Proposed
+
+Supercedes [14. Add storage policies for BlobStore](0014-blobstore-storage-policies.md)
+
+## Context
+
+James exposes a simple BlobStore API for storing raw data. However such raw data often vary in size and access patterns.
+
+As an example:
+
+ - Mailbox message headers are expected to be small and frequently accessed
+ - Mailbox message body are expected to have sizes ranging from small to big but are unfrequently accessed
+ - DeletedMessageVault message headers are expected to be small and unfrequently accessed
+
+The access pattern of some of these kind of blobs do not fit Object Storage caracteristics: good at storing big blobs, but 
```

Comment:
you could define what is "slow". I think observed latency is around 50ms vs 1ms expected from cassandra